### PR TITLE
feat(config): add Cherubini Ora ZRX

### LINKED ISSUE #4459: Missing device configuration: Cherubini ORA ZRX
### Checklist

- [X] It is not in the [configuration DB](https://devices.zwave-js.io/?search=3)

- [X] It was not [merged recently](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Apr+is%3Aclosed) or has a [pending PR](https://github.com/zwave-js/node-zwave-js/pulls?q=is%3Aopen+is%3Apr)

### Which device is missing?

Cherubini Ora ZRX

### Manufacturer ID

0x0455

### Product Type

0x0004

### Product ID

0x0001

### Firmware Version

10.16

### Is the device listed on the Z-Wave Alliance website?

_No response_

### Do you have a manual?

Product manual for this device (motor for awnings, pergolas and screens). Relevant ZWave information from page 65: [ORA_A4518_6N14_EN.pdf](https://github.com/zwave-js/node-zwave-js/files/8466004/ORA_A4518_6N14_EN.pdf)
Product URL: https://www.cherubini-group.com/en/products/motors-8/ora-rx-406

Cherubini has a lot more of ZWave products (https://www.cherubini-group.com/en/products/home-automation-systems-17/meta-320) but I can't see further information of any of them into the ZWave Alliance DB.

Please, find below, debug information from my ZwaveJS2MQTT system. I tried reading "Contributing configuration files" but I'm afraid it is definitely beyond my knowledge.
However, I will try my best if further details would be needed.
```
{
  "id": 25,
  "name": "Awning",
  "loc": "Office",
  "values": [
    {
      "id": "25-38-0-targetValue",
      "nodeId": 25,
      "commandClass": 38,
      "commandClassName": "Multilevel Switch",
      "endpoint": 0,
      "property": "targetValue",
      "propertyName": "targetValue",
      "type": "number",
      "readable": true,
      "writeable": true,
      "label": "Target value",
      "stateless": false,
      "commandClassVersion": 4,
      "min": 0,
      "max": 99,
      "list": false,
      "value": 99,
      "lastUpdate": 1649566471983,
      "newValue": 99
    },
    {
      "id": "25-38-0-duration",
      "nodeId": 25,
      "commandClass": 38,
      "commandClassName": "Multilevel Switch",
      "endpoint": 0,
      "property": "duration",
      "propertyName": "duration",
      "type": "duration",
      "readable": true,
      "writeable": true,
      "label": "Transition duration",
      "stateless": false,
      "commandClassVersion": 4,
      "list": false,
      "value": {
        "value": 0,
        "unit": "seconds"
      },
      "lastUpdate": 1649566471983,
      "newValue": {
        "value": 0,
        "unit": "seconds"
      }
    },
    {
      "id": "25-38-0-currentValue",
      "nodeId": 25,
      "commandClass": 38,
      "commandClassName": "Multilevel Switch",
      "endpoint": 0,
      "property": "currentValue",
      "propertyName": "currentValue",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Current value",
      "stateless": false,
      "commandClassVersion": 4,
      "min": 0,
      "max": 99,
      "list": false,
      "value": 99,
      "isCurrentValue": true,
      "targetValue": "38-0-targetValue",
      "lastUpdate": 1649566471993,
      "newValue": 99
    },
    {
      "id": "25-38-0-Open",
      "nodeId": 25,
      "commandClass": 38,
      "commandClassName": "Multilevel Switch",
      "endpoint": 0,
      "property": "Open",
      "propertyName": "Open",
      "type": "boolean",
      "readable": true,
      "writeable": true,
      "label": "Perform a level change (Open)",
      "ccSpecific": {
        "switchType": 3
      },
      "stateless": false,
      "commandClassVersion": 4,
      "list": false,
      "value": false,
      "lastUpdate": 1649566471993,
      "newValue": false
    },
    {
      "id": "25-38-0-Close",
      "nodeId": 25,
      "commandClass": 38,
      "commandClassName": "Multilevel Switch",
      "endpoint": 0,
      "property": "Close",
      "propertyName": "Close",
      "type": "boolean",
      "readable": true,
      "writeable": true,
      "label": "Perform a level change (Close)",
      "ccSpecific": {
        "switchType": 3
      },
      "stateless": false,
      "commandClassVersion": 4,
      "list": false,
      "value": false,
      "lastUpdate": 1649566471993,
      "newValue": false
    },
    {
      "id": "25-112-0-30",
      "nodeId": 25,
      "commandClass": 112,
      "commandClassName": "Configuration",
      "endpoint": 0,
      "property": 30,
      "propertyName": "param030",
      "type": "number",
      "readable": true,
      "writeable": true,
      "label": "param030 (property)",
      "stateless": false,
      "commandClassVersion": 4,
      "list": false,
      "value": 0,
      "lastUpdate": 1649566471993,
      "newValue": 0
    },
    {
      "id": "25-112-0-31",
      "nodeId": 25,
      "commandClass": 112,
      "commandClassName": "Configuration",
      "endpoint": 0,
      "property": 31,
      "propertyName": "param031",
      "type": "number",
      "readable": true,
      "writeable": true,
      "label": "param031 (property)",
      "stateless": false,
      "commandClassVersion": 4,
      "list": false,
      "value": 0,
      "lastUpdate": 1649566471993,
      "newValue": 0
    },
    {
      "id": "25-112-0-32",
      "nodeId": 25,
      "commandClass": 112,
      "commandClassName": "Configuration",
      "endpoint": 0,
      "property": 32,
      "propertyName": "param032",
      "type": "number",
      "readable": true,
      "writeable": true,
      "label": "param032 (property)",
      "stateless": false,
      "commandClassVersion": 4,
      "list": false,
      "value": 13,
      "lastUpdate": 1649566471993,
      "newValue": 13
    },
    {
      "id": "25-112-0-33",
      "nodeId": 25,
      "commandClass": 112,
      "commandClassName": "Configuration",
      "endpoint": 0,
      "property": 33,
      "propertyName": "param033",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "param033 (property)",
      "stateless": false,
      "commandClassVersion": 4,
      "list": false,
      "value": 0,
      "lastUpdate": 1649566471993,
      "newValue": 0
    },
    {
      "id": "25-112-0-34",
      "nodeId": 25,
      "commandClass": 112,
      "commandClassName": "Configuration",
      "endpoint": 0,
      "property": 34,
      "propertyName": "param034",
      "type": "number",
      "readable": true,
      "writeable": true,
      "label": "param034 (property)",
      "stateless": false,
      "commandClassVersion": 4,
      "list": false,
      "value": 5,
      "lastUpdate": 1649566471993,
      "newValue": 5
    },
    {
      "id": "25-112-0-35",
      "nodeId": 25,
      "commandClass": 112,
      "commandClassName": "Configuration",
      "endpoint": 0,
      "property": 35,
      "propertyName": "param035",
      "type": "number",
      "readable": true,
      "writeable": true,
      "label": "param035 (property)",
      "stateless": false,
      "commandClassVersion": 4,
      "list": false,
      "value": 0,
      "lastUpdate": 1649566471993,
      "newValue": 0
    },
    {
      "id": "25-112-0-36",
      "nodeId": 25,
      "commandClass": 112,
      "commandClassName": "Configuration",
      "endpoint": 0,
      "property": 36,
      "propertyName": "param036",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "param036 (property)",
      "stateless": false,
      "commandClassVersion": 4,
      "list": false,
      "value": 0,
      "lastUpdate": 1649566471993,
      "newValue": 0
    },
    {
      "id": "25-112-0-37",
      "nodeId": 25,
      "commandClass": 112,
      "commandClassName": "Configuration",
      "endpoint": 0,
      "property": 37,
      "propertyName": "param037",
      "type": "number",
      "readable": true,
      "writeable": true,
      "label": "param037 (property)",
      "stateless": false,
      "commandClassVersion": 4,
      "list": false,
      "value": 5,
      "lastUpdate": 1649566471993,
      "newValue": 5
    },
    {
      "id": "25-112-0-38",
      "nodeId": 25,
      "commandClass": 112,
      "commandClassName": "Configuration",
      "endpoint": 0,
      "property": 38,
      "propertyName": "param038",
      "type": "number",
      "readable": true,
      "writeable": true,
      "label": "param038 (property)",
      "stateless": false,
      "commandClassVersion": 4,
      "list": false,
      "value": 0,
      "lastUpdate": 1649566471993,
      "newValue": 0
    },
    {
      "id": "25-113-0-System-Hardware status",
      "nodeId": 25,
      "commandClass": 113,
      "commandClassName": "Notification",
      "endpoint": 0,
      "property": "System",
      "propertyName": "System",
      "propertyKey": "Hardware status",
      "propertyKeyName": "Hardware status",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Hardware status",
      "ccSpecific": {
        "notificationType": 9
      },
      "stateless": false,
      "commandClassVersion": 8,
      "min": 0,
      "max": 255,
      "list": true,
      "states": [
        {
          "text": "idle",
          "value": 0
        },
        {
          "text": "System hardware failure (with failure code)",
          "value": 3
        }
      ],
      "value": 0,
      "lastUpdate": 1649566471993,
      "newValue": 0
    },
    {
      "id": "25-114-0-manufacturerId",
      "nodeId": 25,
      "commandClass": 114,
      "commandClassName": "Manufacturer Specific",
      "endpoint": 0,
      "property": "manufacturerId",
      "propertyName": "manufacturerId",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Manufacturer ID",
      "stateless": false,
      "commandClassVersion": 2,
      "min": 0,
      "max": 65535,
      "list": false,
      "value": 1109,
      "lastUpdate": 1649566471993,
      "newValue": 1109
    },
    {
      "id": "25-114-0-productType",
      "nodeId": 25,
      "commandClass": 114,
      "commandClassName": "Manufacturer Specific",
      "endpoint": 0,
      "property": "productType",
      "propertyName": "productType",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Product type",
      "stateless": false,
      "commandClassVersion": 2,
      "min": 0,
      "max": 65535,
      "list": false,
      "value": 4,
      "lastUpdate": 1649566471993,
      "newValue": 4
    },
    {
      "id": "25-114-0-productId",
      "nodeId": 25,
      "commandClass": 114,
      "commandClassName": "Manufacturer Specific",
      "endpoint": 0,
      "property": "productId",
      "propertyName": "productId",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Product ID",
      "stateless": false,
      "commandClassVersion": 2,
      "min": 0,
      "max": 65535,
      "list": false,
      "value": 1,
      "lastUpdate": 1649566471993,
      "newValue": 1
    },
    {
      "id": "25-117-0-local",
      "nodeId": 25,
      "commandClass": 117,
      "commandClassName": "Protection",
      "endpoint": 0,
      "property": "local",
      "propertyName": "local",
      "type": "number",
      "readable": true,
      "writeable": true,
      "label": "Local protection state",
      "stateless": false,
      "commandClassVersion": 2,
      "list": true,
      "states": [
        {
          "text": "Unprotected",
          "value": 0
        }
      ],
      "value": 0,
      "lastUpdate": 1649566471994,
      "newValue": 0
    },
    {
      "id": "25-117-0-rf",
      "nodeId": 25,
      "commandClass": 117,
      "commandClassName": "Protection",
      "endpoint": 0,
      "property": "rf",
      "propertyName": "rf",
      "type": "number",
      "readable": true,
      "writeable": true,
      "label": "RF protection state",
      "stateless": false,
      "commandClassVersion": 2,
      "list": true,
      "states": [
        {
          "text": "Unprotected",
          "value": 0
        },
        {
          "text": "NoResponse",
          "value": 2
        }
      ],
      "value": 0,
      "lastUpdate": 1649566471994,
      "newValue": 0
    },
    {
      "id": "25-117-0-exclusiveControlNodeId",
      "nodeId": 25,
      "commandClass": 117,
      "commandClassName": "Protection",
      "endpoint": 0,
      "property": "exclusiveControlNodeId",
      "propertyName": "exclusiveControlNodeId",
      "type": "any",
      "readable": true,
      "writeable": true,
      "label": "exclusiveControlNodeId (property)",
      "stateless": false,
      "commandClassVersion": 2,
      "list": false,
      "lastUpdate": 1649566471994
    },
    {
      "id": "25-117-0-timeout",
      "nodeId": 25,
      "commandClass": 117,
      "commandClassName": "Protection",
      "endpoint": 0,
      "property": "timeout",
      "propertyName": "timeout",
      "type": "any",
      "readable": true,
      "writeable": true,
      "label": "timeout (property)",
      "stateless": false,
      "commandClassVersion": 2,
      "list": false,
      "lastUpdate": 1649566471994
    },
    {
      "id": "25-134-0-libraryType",
      "nodeId": 25,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "libraryType",
      "propertyName": "libraryType",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Library type",
      "stateless": false,
      "commandClassVersion": 3,
      "list": true,
      "states": [
        {
          "text": "Unknown",
          "value": 0
        },
        {
          "text": "Static Controller",
          "value": 1
        },
        {
          "text": "Controller",
          "value": 2
        },
        {
          "text": "Enhanced Slave",
          "value": 3
        },
        {
          "text": "Slave",
          "value": 4
        },
        {
          "text": "Installer",
          "value": 5
        },
        {
          "text": "Routing Slave",
          "value": 6
        },
        {
          "text": "Bridge Controller",
          "value": 7
        },
        {
          "text": "Device under Test",
          "value": 8
        },
        {
          "text": "N/A",
          "value": 9
        },
        {
          "text": "AV Remote",
          "value": 10
        },
        {
          "text": "AV Device",
          "value": 11
        }
      ],
      "value": 3,
      "lastUpdate": 1649566471994,
      "newValue": 3
    },
    {
      "id": "25-134-0-protocolVersion",
      "nodeId": 25,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "protocolVersion",
      "propertyName": "protocolVersion",
      "type": "string",
      "readable": true,
      "writeable": false,
      "label": "Z-Wave protocol version",
      "stateless": false,
      "commandClassVersion": 3,
      "list": false,
      "value": "7.16",
      "lastUpdate": 1649566471994,
      "newValue": "7.16"
    },
    {
      "id": "25-134-0-firmwareVersions",
      "nodeId": 25,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "firmwareVersions",
      "propertyName": "firmwareVersions",
      "type": "string[]",
      "readable": true,
      "writeable": false,
      "label": "Z-Wave chip firmware versions",
      "stateless": false,
      "commandClassVersion": 3,
      "list": false,
      "value": [
        "10.16"
      ],
      "lastUpdate": 1649566471994,
      "newValue": [
        "10.16"
      ]
    },
    {
      "id": "25-134-0-hardwareVersion",
      "nodeId": 25,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "hardwareVersion",
      "propertyName": "hardwareVersion",
      "type": "number",
      "readable": true,
      "writeable": false,
      "label": "Z-Wave chip hardware version",
      "stateless": false,
      "commandClassVersion": 3,
      "list": false,
      "value": 1,
      "lastUpdate": 1649566471994,
      "newValue": 1
    },
    {
      "id": "25-134-0-sdkVersion",
      "nodeId": 25,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "sdkVersion",
      "propertyName": "sdkVersion",
      "type": "string",
      "readable": true,
      "writeable": false,
      "label": "SDK version",
      "stateless": false,
      "commandClassVersion": 3,
      "list": false,
      "value": "7.16.2",
      "lastUpdate": 1649566471994,
      "newValue": "7.16.2"
    },
    {
      "id": "25-134-0-applicationFrameworkAPIVersion",
      "nodeId": 25,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "applicationFrameworkAPIVersion",
      "propertyName": "applicationFrameworkAPIVersion",
      "type": "string",
      "readable": true,
      "writeable": false,
      "label": "Z-Wave application framework API version",
      "stateless": false,
      "commandClassVersion": 3,
      "list": false,
      "value": "10.16.2",
      "lastUpdate": 1649566471994,
      "newValue": "10.16.2"
    },
    {
      "id": "25-134-0-applicationFrameworkBuildNumber",
      "nodeId": 25,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "applicationFrameworkBuildNumber",
      "propertyName": "applicationFrameworkBuildNumber",
      "type": "string",
      "readable": true,
      "writeable": false,
      "label": "Z-Wave application framework API build number",
      "stateless": false,
      "commandClassVersion": 3,
      "list": false,
      "value": 267,
      "lastUpdate": 1649566471994,
      "newValue": 267
    },
    {
      "id": "25-134-0-hostInterfaceVersion",
      "nodeId": 25,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "hostInterfaceVersion",
      "propertyName": "hostInterfaceVersion",
      "type": "string",
      "readable": true,
      "writeable": false,
      "label": "Serial API version",
      "stateless": false,
      "commandClassVersion": 3,
      "list": false,
      "value": "unused",
      "lastUpdate": 1649566471994,
      "newValue": "unused"
    },
    {
      "id": "25-134-0-hostInterfaceBuildNumber",
      "nodeId": 25,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "hostInterfaceBuildNumber",
      "propertyName": "hostInterfaceBuildNumber",
      "type": "string",
      "readable": true,
      "writeable": false,
      "label": "Serial API build number",
      "stateless": false,
      "commandClassVersion": 3,
      "list": false,
      "value": 0,
      "lastUpdate": 1649566471994,
      "newValue": 0
    },
    {
      "id": "25-134-0-zWaveProtocolVersion",
      "nodeId": 25,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "zWaveProtocolVersion",
      "propertyName": "zWaveProtocolVersion",
      "type": "string",
      "readable": true,
      "writeable": false,
      "label": "Z-Wave protocol version",
      "stateless": false,
      "commandClassVersion": 3,
      "list": false,
      "value": "7.16.2",
      "lastUpdate": 1649566471994,
      "newValue": "7.16.2"
    },
    {
      "id": "25-134-0-zWaveProtocolBuildNumber",
      "nodeId": 25,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "zWaveProtocolBuildNumber",
      "propertyName": "zWaveProtocolBuildNumber",
      "type": "string",
      "readable": true,
      "writeable": false,
      "label": "Z-Wave protocol build number",
      "stateless": false,
      "commandClassVersion": 3,
      "list": false,
      "value": 267,
      "lastUpdate": 1649566471994,
      "newValue": 267
    },
    {
      "id": "25-134-0-applicationVersion",
      "nodeId": 25,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "applicationVersion",
      "propertyName": "applicationVersion",
      "type": "string",
      "readable": true,
      "writeable": false,
      "label": "Application version",
      "stateless": false,
      "commandClassVersion": 3,
      "list": false,
      "value": "10.16.2",
      "lastUpdate": 1649566471994,
      "newValue": "10.16.2"
    },
    {
      "id": "25-134-0-applicationBuildNumber",
      "nodeId": 25,
      "commandClass": 134,
      "commandClassName": "Version",
      "endpoint": 0,
      "property": "applicationBuildNumber",
      "propertyName": "applicationBuildNumber",
      "type": "string",
      "readable": true,
      "writeable": false,
      "label": "Application build number",
      "stateless": false,
      "commandClassVersion": 3,
      "list": false,
      "value": 43707,
      "lastUpdate": 1649566471994,
      "newValue": 43707
    },
    {
      "id": "25-135-0-80-3",
      "nodeId": 25,
      "commandClass": 135,
      "commandClassName": "Indicator",
      "endpoint": 0,
      "property": 80,
      "propertyName": "Node Identify",
      "propertyKey": 3,
      "propertyKeyName": "On/Off Period: Duration",
      "type": "number",
      "readable": true,
      "writeable": true,
      "description": "Sets the duration of an on/off period in 1/10th seconds. Must be set together with \"On/Off Cycle Count\"",
      "label": "Node Identify - On/Off Period: Duration",
      "ccSpecific": {
        "indicatorId": 80,
        "propertyId": 3
      },
      "stateless": false,
      "commandClassVersion": 3,
      "list": false,
      "value": 0,
      "lastUpdate": 1649566471994,
      "newValue": 0
    },
    {
      "id": "25-135-0-80-4",
      "nodeId": 25,
      "commandClass": 135,
      "commandClassName": "Indicator",
      "endpoint": 0,
      "property": 80,
      "propertyName": "Node Identify",
      "propertyKey": 4,
      "propertyKeyName": "On/Off Cycle Count",
      "type": "number",
      "readable": true,
      "writeable": true,
      "description": "Sets the number of on/off periods. 0xff means infinite. Must be set together with \"On/Off Period duration\"",
      "label": "Node Identify - On/Off Cycle Count",
      "ccSpecific": {
        "indicatorId": 80,
        "propertyId": 4
      },
      "stateless": false,
      "commandClassVersion": 3,
      "list": false,
      "value": 0,
      "lastUpdate": 1649566471995,
      "newValue": 0
    },
    {
      "id": "25-135-0-80-5",
      "nodeId": 25,
      "commandClass": 135,
      "commandClassName": "Indicator",
      "endpoint": 0,
      "property": 80,
      "propertyName": "Node Identify",
      "propertyKey": 5,
      "propertyKeyName": "On/Off Period: On time",
      "type": "number",
      "readable": true,
      "writeable": true,
      "description": "This property is used to set the length of the On time during an On/Off period. It allows asymetic On/Off periods. The value 0x00 MUST represent symmetric On/Off period (On time equal to Off time)",
      "label": "Node Identify - On/Off Period: On time",
      "ccSpecific": {
        "indicatorId": 80,
        "propertyId": 5
      },
      "stateless": false,
      "commandClassVersion": 3,
      "list": false,
      "value": 0,
      "lastUpdate": 1649566471995,
      "newValue": 0
    }
  ],
  "groups": [
    {
      "text": "Lifeline",
      "endpoint": 0,
      "value": 1,
      "maxNodes": 5,
      "isLifeline": true,
      "multiChannel": true
    },
    {
      "text": "Follow me",
      "endpoint": 0,
      "value": 2,
      "maxNodes": 5,
      "isLifeline": false,
      "multiChannel": true
    },
    {
      "text": "Scene activation",
      "endpoint": 0,
      "value": 3,
      "maxNodes": 5,
      "isLifeline": false,
      "multiChannel": true
    },
    {
      "text": "Follow me weather",
      "endpoint": 0,
      "value": 4,
      "maxNodes": 5,
      "isLifeline": false,
      "multiChannel": true
    }
  ],
  "neighbors": [],
  "ready": true,
  "available": true,
  "hassDevices": {},
  "failed": false,
  "inited": true,
  "hexId": "0x0455-0x0004-0x0001",
  "dbLink": "https://devices.zwave-js.io/?jumpTo=0x0455:0x0004:0x0001:10.16",
  "manufacturerId": 1109,
  "productId": 1,
  "productType": 4,
  "productLabel": "Unknown product 0x0001",
  "productDescription": "0x0004",
  "manufacturer": "Unknown manufacturer 0x0455",
  "firmwareVersion": "10.16",
  "protocolVersion": 3,
  "zwavePlusVersion": 2,
  "zwavePlusNodeType": 0,
  "zwavePlusRoleType": 5,
  "nodeType": 1,
  "endpointsCount": 0,
  "endpointIndizes": [],
  "isSecure": true,
  "security": "S2_Authenticated",
  "supportsSecurity": false,
  "supportsBeaming": true,
  "isControllerNode": false,
  "isListening": true,
  "isFrequentListening": false,
  "isRouting": true,
  "keepAwake": false,
  "maxDataRate": 100000,
  "deviceClass": {
    "basic": 4,
    "generic": 17,
    "specific": 7
  },
  "deviceId": "1109-1-4",
  "status": "Alive",
  "interviewStage": "Complete",
  "statistics": {
    "commandsTX": 1,
    "commandsRX": 0,
    "commandsDroppedRX": 0,
    "commandsDroppedTX": 0,
    "timeoutResponse": 0
  },
  "lastActive": 1649566471995,
  "_name": "Office_Awning"
}
```

### PR DESCRIPTION
**Pull request by katiuskt**
My first PR. Trying to contribute to fix https://github.com/zwave-js/node-zwave-js/issues/4459

### CODE REVIEW

File: packages/config/config/devices/0x0455/ora-zrx.json [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+			]
+		},
+		{
+			"#": "32",
+			"label": "Parameter for Window Covering",
+			"description": "Parameter ID used by the window covering set on group Follow Me Weather",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 23,
+			"defaultValue": 13,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0455/ora-zrx.json:73]: Is there an explanation for these values? We try to make the configuration self-explanatory, without having to refer to the manual if possible.

File: packages/config/config/devices/0x0455/ora-zrx.json [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+				}
+			]
+		},
+		{
+			"#": "35",
+			"label": "Sensor Activation Status",
+			"description": "Allows to know which sensor has been associated to the motor. The value of this parameter is the sum of wind, sun and rain status",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 63,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0455/ora-zrx.json]: This looks like it should be split into 3 separate readonly partial parameter, one each for wind, sun and rain status. Does the manual say which bit is which sensor?

**Review by AlCalzone — COMMENTED** [review]
I'm going to give you a more in-depth review tomorrow. E.g. the combinations of label and long description can often be condensed into a different but short label that conveys the same meaning.

For now I have two questions:

File: packages/config/config/devices/0x0455/ora-zrx.json (hunk 1 above)
Comment (katiuskt) [reply packages/config/config/devices/0x0455/ora-zrx.json:73]: I'm no z-wave expert (in fact its terminology is a little bit confusing to me) so I just tried to map this from product guide:
image

File: packages/config/config/devices/0x0455/ora-zrx.json (hunk 2 above)
Comment (katiuskt) [reply packages/config/config/devices/0x0455/ora-zrx.json]: Yes, manual is a little bit more verbose on that but I didn't understand it quite well plus didn't know how to put the information. Any suggestion/explanation is welcome :-)

image

File: packages/config/config/devices/0x0455/ora-zrx.json (hunk 1 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x0455/ora-zrx.json:73]: Ok, that seems to refer to this table (and the following pages) in the Window Covering CC specification:
grafik

We should probably define options for these values with the corresponding labels. I can do that when you're done with the rest.

File: packages/config/config/devices/0x0455/ora-zrx.json (hunk 2 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x0455/ora-zrx.json]: Interesting... assuming the values use the same bits (1+2+4) as in parameter 34, I'm not sure how this is going to go up to 63. Which value does it show on your device?

Comment (katiuskt) [reply packages/config/config/devices/0x0455/ora-zrx.json]: My #35 is 0 but I'm afraid I don't have any sun/wind/rain sensor/device, only the motor itself

image

File: packages/config/config/devices/0x0455/ora-zrx.json [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"description": "Lifeline group",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0455/ora-zrx.json]: Descriptions for association groups are unnecessary and unused by any UI. Please remove them.

File: packages/config/config/devices/0x0455/ora-zrx.json [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+			"label": "Follow Me Weather",
+			"description": "The device in this group will follow the device level if the change is due by the weather condition. The parameter ID sent by this command can be configured by using parameter 32",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "30",
+			"label": "Open Collision Scene",
+			"description": "Scene ID sent if a collision is detected during opening operation",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0455/ora-zrx.json]: As a rule, we try to minimize having descriptions unless they actually add anything. Let's try using "Send Scene ID: Jam When Opening" and removing the description.

File: packages/config/config/devices/0x0455/ora-zrx.json [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+			"options": [
+				{
+					"label": "Do not send the scene activation",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "31",
+			"label": "Close Collision Scene",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0455/ora-zrx.json]: Same: Let's try using "Send Scene ID: Jam When Closing"

File: packages/config/config/devices/0x0455/ora-zrx.json [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+			"label": "Open Collision Scene",
+			"description": "Scene ID sent if a collision is detected during opening operation",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 254,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Do not send the scene activation",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0455/ora-zrx.json]: Change the option label to "Disable"

File: packages/config/config/devices/0x0455/ora-zrx.json [hunk 7]
```diff
@@ -0,0 +1,203 @@
...
+			"label": "Close Collision Scene",
+			"description": "Scene ID sent if a collision is detected during closing operation",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 254,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Do not send the scene activation",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0455/ora-zrx.json]: Change the option label to "Disable"

File: packages/config/config/devices/0x0455/ora-zrx.json [hunk 8]
```diff
@@ -0,0 +1,203 @@
...
+			"description": "Parameter ID used by the window covering set on group Follow Me Weather",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 23,
+			"defaultValue": 13,
+			"unsigned": true
+		},
+		{
+			"#": "33",
+			"label": "Movement Trigger",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0455/ora-zrx.json]: Change label to "Last Movement Trigger" and delete the description.

File: packages/config/config/devices/0x0455/ora-zrx.json [hunk 9]
```diff
@@ -0,0 +1,203 @@
...
+				},
+				{
+					"label": "Rain",
+					"value": 3
+				}
+			]
+		},
+		{
+			"#": "34",
+			"label": "Automation Configuration",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0455/ora-zrx.json]: Change label to "Sun Activation" and make the options "Disable" and "Enable"

File: packages/config/config/devices/0x0455/ora-zrx.json [hunk 10]
```diff
@@ -0,0 +1,203 @@
...
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 63,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": true
+		},
+		{
+			"#": "36",
+			"label": "Lost Life Signal",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0455/ora-zrx.json]: "Signal Connection Status"

File: packages/config/config/devices/0x0455/ora-zrx.json [hunk 11]
```diff
@@ -0,0 +1,203 @@
...
+			"description": "Allows to know if the radio connection is operative or if the motor has lost the connection with its sensors",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": true,
+			"options": [
+				{
+					"label": "Connection is operative",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0455/ora-zrx.json]: Change options to "Active" and "Lost"

File: packages/config/config/devices/0x0455/ora-zrx.json [hunk 12]
```diff
@@ -0,0 +1,203 @@
...
+				},
+				{
+					"label": "Connection has been lost",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "37",
+			"label": "Level Report Period",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0455/ora-zrx.json]: "Level Reporting Frequency" and lose the description.

File: packages/config/config/devices/0x0455/ora-zrx.json [hunk 13]
```diff
@@ -0,0 +1,203 @@
...
+			"valueSize": 2,
+			"minValue": 2,
+			"maxValue": 60,
+			"defaultValue": 5,
+			"unsigned": true
+		},
+		{
+			"#": "38",
+			"label": "Send Multilevel Report",
+			"description": "For backward compatibility: device can send the multilevel report together with the switch multilevel report in addition to the update with windows covering report",
```
Comment (blhoward2) [inline packages/config/config/devices/0x0455/ora-zrx.json]: Unnecessary description. Change the option labels to "Enable" and "Disable"

**Review by blhoward2 — DISMISSED** [review]
Great job on this for your first time. In addition to Al's comments, I went ahead and did a review to our style guide. Please go ahead and make the changes to clarify the descriptions, remove the unnecessary descriptions, and conform the option labels so they are shorter.

File: packages/config/config/devices/0x0455/ora-zrx.json (hunk 3 above)
Comment (katiuskt) [reply packages/config/config/devices/0x0455/ora-zrx.json]: Done

File: packages/config/config/devices/0x0455/ora-zrx.json (hunk 4 above)
Comment (katiuskt) [reply packages/config/config/devices/0x0455/ora-zrx.json]: Thanks, understood

File: packages/config/config/devices/0x0455/ora-zrx.json (hunk 5 above)
Comment (katiuskt) [reply packages/config/config/devices/0x0455/ora-zrx.json]: Done

File: packages/config/config/devices/0x0455/ora-zrx.json (hunk 2 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x0455/ora-zrx.json]: Ok, we'll leave this as is until someone comes who knows more.

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
Thanks a lot!

**Pull request merged**